src/components/atoms/LamasLogo.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC, useMemo } from 'react';
+import LamasImage from '../../assets/cbs.png';
+
+interface LamasLogoProps {
+  height: string;
+}
+
+export const LamasLogo: FC<LamasLogoProps> = ({ height }) => {
+  const logo = useMemo(() => LamasImage.toString(), []);
+
+  const logoStyle = useMemo(
```

Comment:
Why [memoize](https://reactjs.org/docs/hooks-reference.html#usememo) a `prop`?
Make sure you understand the hook before using it :)

Reply:
I've duplicate the AnywayLogo component and adjust it to the LamasLogo.